### DISCUSSION TITLE
Workflow suggestions regarding single hand use

### DISCUSSION THREAD

**Discussion by azzamsa** [manna-harbour/miryoku#344] [Q&A] [answered]
Hi,

When I'm editing videos, I use my right hand on the mouse extensively. However, when I need to delete a clip repeatedly, I have to move my hand from the mouse to the `delete` key, which is inconveniently located on the right side of the keyboard.

If there were a way to activate a `single-hand mode` that replicates the delete key on the left side of the keyboard, I wouldn't have to move my hand at all.

Do you have any suggestions for this?

---

A happy Miryoku user for over 3 years. 🥞

**Comment by manna-harbour** [comment, marked as answer]
There are a few options:

- Add it to an unused key on the same hand as a layer hold.
  - E.g. holding left home thumb for Nav, you have mods on left hand home fingers, but the other keys are unused (https://github.com/manna-harbour/miryoku/tree/master/docs/reference#nav). Other than double taps for Additional Features (https://github.com/manna-harbour/miryoku/tree/master/docs/reference#additional-features), but they could be disabled if they interfere.
- Add a custom layer for one hand use that can be activated with one hand.
  - E.g. substitute custom Extra or Tap layers (https://github.com/manna-harbour/miryoku/tree/master/docs/reference#additional-features).
- Replace some keys on the Button layer (https://github.com/manna-harbour/miryoku/tree/master/docs/reference#button).
- Add a combo to Base.
- Map the extra keys if your board has more than 3x5+3.
- Enable QMK swap hands (https://docs.qmk.fm/features/swap_hands).
- Use a macro pad.
- Use a pointing device on the left.

See https://github.com/manna-harbour/miryoku/discussions/85 for customisation.

**Reply by azzamsa** [reply]
Thank you for very thorough information! 🥞
